torch_dftd/nn/dftd3_module.py

```diff
@@ -76,27 +76,29 @@ def calc_energy_batch(
         damping: str = "zero",
     ) -> Tensor:
         """Forward computation to calculate atomic wise dispersion energy"""
+        shift = pos.new_zeros((edge_index.size()[1], 3, 3)) if shift is None else shift
         pos_bohr = pos / d3_autoang  # angstrom -> bohr
         if cell is None:
-            cell_bohr = None
+            cell_bohr: Optional[Tensor] = None
```

Comment:
Because of the change of `calc_distances()`, `cell` is no longer used within `dftd3_module`, but I kept it just in case.
Same things applied to `dftd2_module`.